### PR TITLE
css layout style

### PR DESCRIPTION
**Pull request by sergiomons**

### CODE REVIEW

**Review by abelhernando — COMMENTED** [review]
GOOD JOB! Keep it that way!! :D 🗡

**Comment by PacoGD** [comment]
Well done!

File: staff/sergiomons/css-layout-simple-adaptive/styles/main.css [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@import "mobile.css" screen and (max-width: 768px);
+@import "desktop.css" screen and (min-width: 768px);
```
Comment (agandia9) [inline staff/sergiomons/css-layout-simple-adaptive/styles/main.css]: What are the common properties that are repeated in both files (mobile and desktop)? Should you group them here?

File: staff/sergiomons/css-layout-simple-cv/styles/main.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    background-color: white;
+}
+header{
+  width:100%;
+  height: 20%;
+  border-bottom: 2px solid black;
+  text-align: center;
+  
+  
+}
```
Comment (agandia9) [inline staff/sergiomons/css-layout-simple-cv/styles/main.css]: Try to organize your css following an ordered structure,
first the generic selectors by <tag>, then the class and finally the #id, the latter being the most specific.

[Check this, 5 point](https://code.tutsplus.com/tutorials/30-css-best-practices-for-beginners--net-6741)

File: staff/sergiomons/css-layout-simple-adaptive/styles/desktop.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
+* {
```
Comment (manuelbarzi) [inline staff/sergiomons/css-layout-simple-adaptive/styles/desktop.css]: TIP group common css rules (mobile and desktop) in main.css

File: staff/sergiomons/css-layout-simple-adaptive/styles/desktop.css [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+ }
+ html, body {
+    min-height: 100% !important;
+    height: 100%;
+    background-color: black;
+    color:white;
+    text-align:center;
+}
+header, footer{
+  width:100%;
```
Comment (manuelbarzi) [inline staff/sergiomons/css-layout-simple-adaptive/styles/desktop.css]: TIP is ```width: 100%```rule required?

File: staff/sergiomons/css-layout-simple-adaptive/styles/desktop.css [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+nav, aside{
+   width:20%;
+   float: left;
+   height: 100%;
+   border: 1px solid white;
+}
+section {
+    width: 60%;
+    float: left;
+    height: 100%;
```
Comment (manuelbarzi) [inline staff/sergiomons/css-layout-simple-adaptive/styles/desktop.css]: TIP group selectors for equal rules.

SEE Grouping Selectors https://www.w3schools.com/css/css_syntax.asp

File: staff/sergiomons/css-layout-simple/styles/main.css [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    height: 25%;
+    background-color: bisque;
+}
+main {
+    height: 50%;
+    width: 100%;
+}
+aside {
+    height: 100%;
+    float: left;
```
Comment (manuelbarzi) [inline staff/sergiomons/css-layout-simple/styles/main.css:20]: TIP group selectors for same rules

File: staff/sergiomons/css-layout-simple-adaptive/styles/main.css (hunk 1 above)
Comment (sergiomons) [reply staff/sergiomons/css-layout-simple-adaptive/styles/main.css]: Yes, you're right! Thanks!

File: staff/sergiomons/css-layout-simple-adaptive/styles/desktop.css (hunk 3 above)
Comment (sergiomons) [reply staff/sergiomons/css-layout-simple-adaptive/styles/desktop.css]: Ok, I,ll change it, thanks!

File: staff/sergiomons/css-layout-simple-adaptive/styles/desktop.css (hunk 4 above)
Comment (sergiomons) [reply staff/sergiomons/css-layout-simple-adaptive/styles/desktop.css]: Ok, I think it's not necessary. I,m going to delete it.

File: staff/sergiomons/css-layout-simple-adaptive/styles/desktop.css (hunk 5 above)
Comment (sergiomons) [reply staff/sergiomons/css-layout-simple-adaptive/styles/desktop.css]: OK, I,m going to group them!

File: staff/sergiomons/js/challenges/to-camel-case/scripts/to-camel-case.js [hunk 7]
```diff
@@ -0,0 +1,9 @@
+'use strict'
```
Comment (manuelbarzi) [inline staff/sergiomons/js/challenges/to-camel-case/scripts/to-camel-case.js]: TIP ; at end of statement

File: staff/sergiomons/js/challenges/to-camel-case/scripts/main.js [hunk 8]
```diff
@@ -0,0 +1,4 @@
+'use strict'
```
Comment (manuelbarzi) [inline staff/sergiomons/js/challenges/to-camel-case/scripts/main.js]: TIP ; at end of statement

File: staff/sergiomons/js/challenges/count-word/scripts/main.js [hunk 9]
```diff
@@ -0,0 +1,3 @@
+var count = countChart("hello world")
```
Comment (manuelbarzi) [inline staff/sergiomons/js/challenges/count-word/scripts/main.js]: TIP 'use strict';

Comment (manuelbarzi) [reply staff/sergiomons/js/challenges/count-word/scripts/main.js]: WARN ```countChart```, apart from being a wrong english word  (is not ```chart```, but ```char```), is not the correct function name. the correct function name is ```countWords```. rename it.

folder name is also incorrect. rename it to ```count-words```

File: staff/sergiomons/js/challenges/count-word/scripts/count-word.js [hunk 10]
```diff
@@ -0,0 +1,6 @@
+function countChart(str) {
```
Comment (manuelbarzi) [inline staff/sergiomons/js/challenges/count-word/scripts/count-word.js]: TIP 'use strict';

Comment (manuelbarzi) [reply staff/sergiomons/js/challenges/count-word/scripts/count-word.js]: WARN ```countChart```is incorrect. the correct function name is ```countWords````. rename it.

Comment (manuelbarzi) [reply staff/sergiomons/js/challenges/count-word/scripts/count-word.js]: WARN file name is incorrect. rename it to ```countWords.js``` instead.

Comment (sergiomons) [reply staff/sergiomons/js/challenges/count-word/scripts/count-word.js]: Ok, I,m rename it. Thanks!

File: staff/sergiomons/css-layout-simple-adaptive/styles/main.css (hunk 1 above)
Comment (sergiomons) [reply staff/sergiomons/css-layout-simple-adaptive/styles/main.css]: I have modified it.

File: staff/sergiomons/js/challenges/count-word/scripts/main.js (hunk 9 above)
Comment (sergiomons) [reply staff/sergiomons/js/challenges/count-word/scripts/main.js]: I have changed it. Thanks.

File: staff/sergiomons/js/challenges/to-camel-case/scripts/main.js (hunk 8 above)
Comment (sergiomons) [reply staff/sergiomons/js/challenges/to-camel-case/scripts/main.js]: I have changed it. Thanks.

File: staff/sergiomons/js/challenges/to-camel-case/scripts/to-camel-case.js (hunk 7 above)
Comment (sergiomons) [reply staff/sergiomons/js/challenges/to-camel-case/scripts/to-camel-case.js]: I have changed it. Thanks.

File: staff/sergiomons/app-template/landing.html [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+        <header>
+           <h2>FitFood💪</h2>
+        </header>
+        <div class="p-intro">
+            <p>¿Why not be wealthy and eat it at home?</p>                  
+        </div>
+        <div class="btns">
+              <a href=#login.html class="buttons">Login</a>
+              <a href=#register.html class="buttons">Register</a>
+           <!--  <input type="button" name="Login" value="Login" id="login" class="buttons">
```
Comment (agandia9) [inline staff/sergiomons/app-template/landing.html]: delete comments before push!

File: staff/sergiomons/app-template/styles/landing.css [hunk 12]
```diff
@@ -0,0 +1,90 @@
+@import "common1.css";
+.content {
+    display: grid;
+    height: 100%;
+    /*grid-template-columns: repeat(auto-fit, minmax(40%, 1fr));*/
```
Comment (agandia9) [inline staff/sergiomons/app-template/styles/landing.css]: !!

File: staff/sergiomons/app-template/styles/landing.css [hunk 13]
```diff
@@ -0,0 +1,90 @@
...
+    display: flex;
+    justify-content: center;
+    grid-row: 4;
+}
+.content > .p-regist {
+    grid-row: 5;
+    font-size: 130%;
+}
+
+/*.buttons {
```
Comment (agandia9) [inline staff/sergiomons/app-template/styles/landing.css]: !!!!!!

Comment (sergiomons) [reply staff/sergiomons/app-template/styles/landing.css]: Jejeje yes, i forget delete them. I have done it!

**Pull request merged**